pkgs/tools/misc/tvnamer/default.nix

```diff
@@ -1,5 +1,6 @@
 { lib
 , python3
+, python3Packages
```

Comment:
```suggestion
```

Reply:
Thanks, I used this change and is part of the new version.